model-client/integration-tests/src/commonTest/kotlin/org/modelix/model/client2/MockServerUtils.kt

```diff
@@ -0,0 +1,46 @@
+package org.modelix.model.client2
+
+import io.ktor.client.HttpClient
+import io.ktor.client.plugins.expectSuccess
+import io.ktor.client.request.put
+import io.ktor.client.request.setBody
+import io.ktor.http.ContentType
+import io.ktor.http.appendPathSegments
+import io.ktor.http.contentType
+import io.ktor.http.takeFrom
+
+/**
+ * Wrapper to interact with the [mock-server](https://www.mock-server.com)
+ * started by model-client-integration-tests/docker-compose.yaml
+ *
+ * Uses the REST API instead of JS and JAVA SDKs to be usable with multiplatform tests.
+ * See https://app.swaggerhub.com/apis/jamesdbloom/mock-server-openapi
+ */
+object MockServerUtils {
+    // The port was chosen random, but is static.
```

Comment:
```suggestion
    // The port was chosen randomly, but is static.
```

I don't get what this sentence should mean.

Reply:
I updatd the comments here.
Maybe it is now more clear.